Created Output Log File: /workdir/tasks_run/data/nfb_logs/p004_calculator_script_20240912_14h09m08s.txtexperimenter screen width: 3390
experimenter screen height: 1080
Second monitor resolution: 1920x1080
TO SHOW INSTRUCTIONS, PLEASE PRESS 'r'.
Showing Instructions. Task will start when 's' is pressed (scanner key maps to s on mac, so starting the experiment will trigger the task)
Running Main Calculation Script ... 
Starting Block1 ... 
========================================
Starting Block1, Trial 1... 
========================================
Waiting For New File ...
New File Found In Dir...
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/4956873936264875512.dcm
Mean ROI Activation: 185.88189697265625
length design matrix: 1
  trial_type  onset  duration
0       rest    0.0      1.06
Concatenating 1 images.
Error:
File not found: '/workdir/tasks_run/data/roi_masks/nii_TR1.nii_zipped'
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 28, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 26, in run_trial
    dictionary = Calculator.get_resid(dictionary=dictionary, block=block, trial=trial)
  File "/workdir/tasks_run/scripts/Calculator.py", line 108, in get_resid
    concatNii = concat_imgs(niiList)
  File "/usr/local/lib/python3.10/site-packages/nilearn/_utils/niimg_conversions.py", line 490, in concat_niimgs
    first_niimg = check_niimg(next(literator), ensure_ndim=ndim)
  File "/usr/local/lib/python3.10/site-packages/nilearn/_utils/niimg_conversions.py", line 303, in check_niimg
    raise ValueError(f"File not found: '{niimg}'")
ValueError: File not found: '/workdir/tasks_run/data/roi_masks/nii_TR1.nii_zipped'

Retries left: 3
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/4956873936264875512.dcm
Mean ROI Activation: 185.88189697265625
length design matrix: 1
  trial_type  onset  duration
0       rest    0.0      1.06
Concatenating 2 images.
Error:
File not found: '/workdir/tasks_run/data/roi_masks/nii_TR1.nii_zipped'
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 28, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 26, in run_trial
    dictionary = Calculator.get_resid(dictionary=dictionary, block=block, trial=trial)
  File "/workdir/tasks_run/scripts/Calculator.py", line 108, in get_resid
    concatNii = concat_imgs(niiList)
  File "/usr/local/lib/python3.10/site-packages/nilearn/_utils/niimg_conversions.py", line 490, in concat_niimgs
    first_niimg = check_niimg(next(literator), ensure_ndim=ndim)
  File "/usr/local/lib/python3.10/site-packages/nilearn/_utils/niimg_conversions.py", line 303, in check_niimg
    raise ValueError(f"File not found: '{niimg}'")
ValueError: File not found: '/workdir/tasks_run/data/roi_masks/nii_TR1.nii_zipped'

Getting New Dicom Instead...
Waiting For New File ...
New File Found In Dir...
Retries left: 2
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/12855096946696320955.dcm
Mean ROI Activation: 185.88189697265625
length design matrix: 1
  trial_type  onset  duration
0       rest    0.0      1.06
Concatenating 3 images.
Error:
File not found: '/workdir/tasks_run/data/roi_masks/nii_TR1.nii_zipped'
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 28, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 26, in run_trial
    dictionary = Calculator.get_resid(dictionary=dictionary, block=block, trial=trial)
  File "/workdir/tasks_run/scripts/Calculator.py", line 108, in get_resid
    concatNii = concat_imgs(niiList)
  File "/usr/local/lib/python3.10/site-packages/nilearn/_utils/niimg_conversions.py", line 490, in concat_niimgs
    first_niimg = check_niimg(next(literator), ensure_ndim=ndim)
  File "/usr/local/lib/python3.10/site-packages/nilearn/_utils/niimg_conversions.py", line 303, in check_niimg
    raise ValueError(f"File not found: '{niimg}'")
ValueError: File not found: '/workdir/tasks_run/data/roi_masks/nii_TR1.nii_zipped'

Getting New Dicom Instead...
Waiting For New File ...
New File Found In Dir...
Retries left: 1
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/16350206997696942276.dcm
---- Keyboard Interrupt Detected ----
What Would You Like to Do?
(1) Continue With The Block
(2) End The Session
(3) Start New Block
dicom dir using: /workdir/tasks_run/data/sambashare/test_dir
